Runtime/AvatarCreator/Scripts/AvatarTemplateFetcher.cs

```diff
@@ -7,17 +7,28 @@
 
 namespace ReadyPlayerMe.AvatarCreator
 {
+    public enum TemplateVersions
+    {
+        All,
+        V1,
+        V2
+    }
+
     /// <summary>
     /// This class can be used to fetch avatar template data including icon renders from the avatarAPI.
     /// </summary>
     public class AvatarTemplateFetcher
     {
         private readonly CancellationToken ctx;
         private readonly AvatarAPIRequests avatarAPIRequests;
+        private const string TEMPLATE_ONBOARDING_USAGE_TYPE = "onboarding";
+        private const string TEMPLATE_RANDOMIZE_USAGE_TYPE = "randomize";
```

Comment:
I dont understand the naming of it, but would it make sense to rename the variables to 

TEMPLATE_V1_USAGE_TYPE and TEMPLATE_V2_USAGE_TYPE. Onboarding and Randomize don't say much to be fair :D .